src/Dropdown/Dropdown.tsx

```diff
@@ -117,12 +119,22 @@ export const Dropdown = forwardRef(function Dropdown(
         >
           {hasOptGroups ? (
             <optgroup label={placeholder}>
-              <option value="" hidden>
+              <option
+                value=""
```

Comment:
Suggestion: can this just be hidden={!(placeholder && showPlaceholderAsValue)}

Reply:
The actual property is just `hidden` so any existence of it will trigger it to be hidden. adding `true` or `false` makes no difference so we need to when we don't want the `hidden` attribute, we need to entirely remove the prop.